shared/templates/mount_option/bash.template

```diff
@@ -6,8 +6,8 @@
 include_mount_options_functions
 
 function perform_remediation {
-	# test "$mount_has_to_exist" = 'yes'
-	if test "{{{ MOUNT_HAS_TO_EXIST }}}" = 'yes'; then
+	# test "$mount_has_to_exist" = 'no'
+	if test "{{{ MOUNT_HAS_TO_EXIST }}}" = 'no'; then
```

Comment:
you should also flip the condition, right?

Reply:
Ah, d'oh. Yes.